app/NavEntry.js

```diff
@@ -29,6 +34,27 @@ class MainNavigation extends React.Component {
     }
   }
 
+  async componentDidMount() {
+    const { t } = this.props;
+    try {
+      const { data } = await fetch(BULLETINS_URL);
+      const bulletins = data.bulletins;
+      const lastBulletin = await GetStoreData('lastBulletin', true);
```

Comment:
I think this should be newBulletin or newBulletinAvailable instead of lastBulletin
```suggestion
      const lastBulletin = await GetStoreData('newBulletin', true);
```